core/controllers/editor.py

```diff
@@ -607,7 +581,8 @@ def get(self, exploration_id, escaped_state_name):
         # matched to a fuzzy rule individually.
         answers = stats_services.get_top_state_rule_answers(
             exploration_id, state_name, [
-                exp_domain.DEFAULT_RULESPEC_STR, rule_domain.FUZZY_RULE_TYPE],
+                exp_domain.DEFAULT_OUTCOME_CLASSIFICATION,
+                exp_domain.TRAINING_DATA_CLASSIFICATION],
```

Comment:
Why include this one? From the docstring for TRAINING_DATA_CLASSIFICATION, I thought these are answers that are the training data for an answer group?

Also, should some of the TODOs above be removed?